PhoneNumbers/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file related to the two samples?

Reply:
No, it's a different sample pending from some time. Missed to merge.